2017/07/01_playing_with_time.py

```diff
@@ -88,6 +95,10 @@ def string_to_date(date_string: str) -> date:
 def is_future(dt: datetime) -> bool:
     """Return whether the passed datetime is in the future"""
 
+    if dt > datetime.now():
+        return True
+    else:
+        return False
```

Comment:
Because `dt > datetime.now()` returns a boolean, you can also just `return dt > datetime.now()`